### PR TITLE
feat(ArcViewer): add presence

### PR DESCRIPTION
**Pull request by joerkig**
## Description 

First time using getLogs please help me improve if needed. did test leaving page open on new extension version did not seem to cause slowdowns even after leaving open for more than 10 minutes when the old version would've, maybe don't merge until new extension version is on chrome?

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

image
image

</details>

### CODE REVIEW

**Comment by theusaf** [comment]
Marking as draft. Currently, `getLogs` is a bit unstable, waiting for updates to extension first. You may continue making changes to this PR.

File: websites/A/ArcViewer/presence.ts [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+			presence.getSetting<boolean>("cover"),
+			presence.getSetting<number>("mapSmallImages"),
+		]),
+		presenceData: PresenceData = {
+			largeImageKey: Assets.Logo,
+			startTimestamp: browsingTimestamp,
+		},
+		logs = await presence.getLogs();
+
+	for (const line of logs) {
```
Comment (theusaf) [inline websites/A/ArcViewer/presence.ts:96]: Make sure to merge/rebase the latest main into your branch.

Either:
* Add type check for string
* Assume strings and use string generic with `getLogs`

**Review by theusaf — CHANGES_REQUESTED** [review]

**Comment by Bas950** [comment]
The getLogs issues should be fixed in the extension update that was released, can you please check @joerkig

**Review by theusaf — APPROVED** [review]
Images look good

**Review by Bas950 — APPROVED** [review]

**Pull request merged**